inc/saitypes.h

```diff
@@ -463,6 +465,9 @@ typedef struct _sai_tunnel_map_params_t
     /** VNI id */
     sai_uint32_t vni_id;
 
+    /** Bridge IF */
+    sai_object_id_t bridge_if;
```

Comment:
does this needs to be here ? it will make really hard to track this object id, there is already oid in attribute_t directly in the structure